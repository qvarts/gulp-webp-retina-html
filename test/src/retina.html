<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Retina images</title>
</head>
<body>
<img class="test-class" src="/img/test_retina.jpg" alt="test" />
</body>
</html>