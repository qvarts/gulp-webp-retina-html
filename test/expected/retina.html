<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Retina images</title>
</head>
<body>
<picture>
<source srcset="/img/test_retina.jpg 1x, /img/test_retina@2x.jpg 2x, /img/test_retina@3x.jpg 3x, /img/test_retina@4x.jpg 4x" type="image/jpeg">
<img class="test-class" src="/img/test_retina.jpg" alt="test" />
</picture>
</body>
</html>